plugin/storage/kafka/options.go

```diff
@@ -108,9 +108,9 @@ var (
 
 // Options stores the configuration options for Kafka
 type Options struct {
-	config   producer.Configuration
-	topic    string
-	encoding string
+	Config   producer.Configuration `mapstructure:",squash"`
```

Comment:
Does this need `mapstructure:"producer....`?

Reply:
I was thinking about it, but given it is scoped under exporters it does not make much sense.